cookbook/core/extend_flyte/sql.py

```diff
@@ -0,0 +1,79 @@
+"""
+.. _extend_sql_sqlite3:
+
+SQL Query Tasks - example SQLite3
+===================================
+
+Flyte tasks are not restricted to pure containers. This is one of the most important design decisions in Flyte.
+Non-container tasks can have arbitrary targets for execution - example an API that executes SQL queries like SnowFlake,
+BigQuery etc, a synchronous WebAPI etc
+
+The follow example shows how you can write SQLite3 queries using the SQLite3Task, which is bundled as part of the core
+flytekit. Since SQL Queries are portable across SQL environments (as long as the data exists), users are not required
```

Comment:
```suggestion
flytekit. Since SQL Queries are portable across workflows and Flyte installations (as long as the data exists), this task will always run with a pre-built container, specifically the [flytekit container](https://github.com/flyteorg/flytekit/blob/v0.19.0/Dockerfile.py38) itself.
```